docs/snapshotting/random-for-clones.md

```diff
@@ -53,12 +58,13 @@ multiple parameters (such as timer data, or output from `CPU HWRNG`
 instructions which are present on Ivy Bridge or newer Intel processors
 and enabled in a Firecracker guest) are mixed with each result. Extra
 bits are mixed in both when reading random values, and in conjunction
-with entropy related events such as interrupts. There are two questions
-here: is the `CPU HWRNG` output always mixed in when the feature is
-present (as opposed to only when the `CPU HWRNG` is trusted), and is the
+with entropy related events such as interrupts. Moreover, the guest kernel
+will eventually receive fresh entropy from `virtio-rng`, if attached. There are
```

Comment:
nit: this sentence is rather long and hard to fit into cache. Is it possible to organise it somehow, maybe via bullet points for the questions?

Reply:
I agree. I re-worked this. Please take a look.